ga4gh/frontend.py

```diff
@@ -397,6 +400,17 @@ def checkAuthentication():
         else:
             return startLogin()
 
+    jsonBody = flask.request.get_json() or {}
```

Comment:
This is a definite break in layering, which is a bad idea. What we have here is fine for a proof of concept, but it's too easy to break this for real use. I'll outline a different approach that we might use below though.